julia-editor/index.html

```
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+    <meta charset="UTF-8" />
+    <meta name>="viewport" content="width=device-width,initial-scale=1.0"/>
+    <title>Julia Editor</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <div class=""container">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)